releasenotes/notes/snmp_traps_remove_dots-80e39de3aa965fa9.yaml

```diff
@@ -0,0 +1,11 @@
+# Each section from every release note are combined when the
+# CHANGELOG.rst is rendered. So the text needs to be worded so that
+# it does not depend on any information only available in another
+# section. This may mean repeating some details, but each section
+# must be readable independently of the other.
+#
+# Each section note must be formatted as reStructuredText.
+---
+enhancements:
+  - |
+   [snmp] Remove initial dot for OID in traps variables.
```

Comment:
I think there is no need to have a release note for this PR since it's a bug fix.

About the CI job checking for release note, you can bypass it by adding this label to the PR `changelog/no-changelog`